.circleci/config.yml

```diff
@@ -68,6 +68,9 @@ commands:
 
             # Install node dependencies
             npm ci
+            
+            # Symlink local dependencies
```

Comment:
I think we need to be transparent about why we use this command. That we are fixing a bug that manifests itself on windows. That way we have some idea to come back and remove it if we fix the source of the problem.